app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,7 @@
     <string name="welcome_message">Welcome to NoteX</string>
     <string name="retry">Retry</string>
     <string name="empty_message">It\'s all empty here</string>
+    <string name="cd_add">Add</string>
```

Comment:
Please mention that `cd` standas for content description. everyone might not get it directly.

Reply:
okayy